pkg/zeek/addr.go

```diff
@@ -56,6 +58,11 @@ func (a *Addr) Type() Type {
 	return TypeAddr
 }
 
+func (a *Addr) Encode(dst zval.Encoding) zval.Encoding {
+	b := []byte(a.Native.String())
```

Comment:
Don't have to address this in this pr, but it would be better if this didn't have to do the alloc here.

Reply:
This is a deliberate placeholder till we switch to machine types

Reply:
That's what I thought, cool